test/junit/src/test/scala/strawman/collection/mutable/StringBuilderTest.scala

```diff
@@ -0,0 +1,24 @@
+package strawman.collection.mutable
+
+import org.junit.Assert.assertEquals
+import strawman.collection.immutable.List
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.junit.Test
+
+import scala.tools.testing.AssertUtil
+
+@RunWith(classOf[JUnit4])
+class StringBuilderTest {
+  @Test
+  def specificBuilder: Unit = {
+    val b = new StringBuilder() ++= "abcd"
+    val b1 = b.filter(_ % 2 == 0)
+    val b1t: StringBuilder = b1
+    assertEquals(b1t.toString, "bd")
+
+    val b2 = b.map(c => (c + 1).toChar)
+    val b2t: IndexedSeq[Char] = b2
+    assertEquals(b2t.toString, "ArrayBuffer(b, c, d, e)")
```

Comment:
to improve this i'd have to add an overload for `map` i guess? what other operations have to be added manually in for such collections? `flatMap`, ...? (how would a user / collection implementer find out?)

Reply:
for the record, the behavior as tested here is the same as in 2.12

Reply:
> what other operations have to be added manually in for such collections? flatMap, ...? (how would a user / collection implementer find out?)

All the operations that return a `CC[B]`. I’ve just had a look and several of them are missing overloads (e.g. `scanLeft`)…

I’ve added the following entry in the FAQ: https://github.com/scala/collection-strawman/wiki/FAQ#which-methods-should-i-overload-to-support-the-same-result-type-principle